frontend/README.md

```diff
@@ -49,34 +53,37 @@ This is the easiest way to start developing, but it does not support all apis du
 development.
 
 Run `npm run mock:api` to start a mock backend api server handler so it can
-serve basic api calls with mock data. 
+serve basic api calls with mock data.
 
 ### Proxy to a real cluster
 
 This requires you already have a real KFP cluster, you can proxy requests to it.
 
-Before you start, configure your `kubectl` to talk to your KFP cluster.
+1. [Install Kubeflow Pipelines](https://www.kubeflow.org/docs/pipelines/installation/overview/) based on your use case and environment.
+1. Configure your `kubectl` to [access your KFP cluster](https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl).
```

Comment:
Note, the command is only for GCP. Other developers may be using their own.

Reply:
Thank you for explaining! Since it is hard to list out all the possible steps for connect kubectl the desired cluster, I have updated the description to call out that this link is for GCP only.